src/app/components/edit-nonce-button.tsx

```diff
@@ -1,12 +1,12 @@
-import { ButtonProps, LeatherButton } from '@app/ui/components/button';
+import { Link, LinkProps } from '@app/ui/components/link/link';
 
-interface EditNonceButtonProps extends ButtonProps {
+interface EditNonceButtonProps extends LinkProps {
   onEditNonce(): void;
 }
 export function EditNonceButton({ onEditNonce, ...props }: EditNonceButtonProps) {
   return (
-    <LeatherButton onClick={onEditNonce} textStyle="label.02" variant="link" {...props}>
+    <Link onClick={onEditNonce} {...props}>
```

Comment:
Maybe we can rename `EditNonceButton` now it's actually a link? 

Or even better get rid of it and replace it with `<Link`

Reply:
do it

Reply:
Yeah, I'll try to go back thru this today to clean up where the new components are now used. I didn't spend much time evaluating that part of this work yet. Good call here.

Reply:
Thanks 👍 we can fix stuff like that on subsequent smaller PRs, just mentioning as I notice it